wrangler-core/src/main/java/io/cdap/wrangler/metrics/Constants.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.wrangler.metrics;
+
+/**
+ * Constants for emitting CDAP metrics from Wrangler
+ */
+public class Constants {
+  public static final String APPLICATION_NAME = "dataprep";
```

Comment:
this shouldn't be used, the app name is something only the platform knows, not the plugin.

Reply:
It's already defined in the wrangler-transform package that has the Wrangler plugin class as well though: https://github.com/data-integrations/wrangler/blob/develop/wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java#L101

Reply:
that's the application name for the service (also it looks unused in that class?). Application name for a pipeline is the pipeline name, which is not a constant.